app/styles/modules/_tooltip.scss

```diff
@@ -2,7 +2,12 @@
   background: $error-background-color;
   border-radius: $small-border-radius;
   color: $message-text-color;
-  left: 3px;
+  html[dir=ltr] & {
+  	left: 3px;
+  	}
```

Comment:
Can you fix the indentation here? The closing brace should be left aligned with the selector. a two space indent, and a blank line above the entire declaration:

```css
color: $message-text-color;
html[dir=ltr] & {
  left: 3px;
}
```

You can see where there are issues by running `grunt sasslint` at the command line from the root directory.